src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -258,24 +250,32 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
       />
       {hasSnapshots ? (
         <>
-          <SelectableMainTable
-            headers={headers}
-            rows={pagination.pageData}
-            sortable
-            className="snapshots-table"
-            id="snapshots-table-wrapper"
-            emptyStateMsg="No snapshot found matching this search"
-            itemName="snapshot"
-            parentName="instance"
-            selectedNames={selectedNames}
-            setSelectedNames={setSelectedNames}
-            processingNames={processingNames}
-            totalCount={instance.snapshots?.length ?? 0}
-            filteredNames={filteredSnapshots.map((snapshot) => snapshot.name)}
-            onUpdateSort={pagination.updateSort}
-            defaultSort="created_at"
-            defaultSortDirection="descending"
-          />
+          <ScrollableTable
+            dependencies={[
+              instance.snapshots,
+              inTabNotification,
+              query,
+              pagination.pageSize,
+              pagination.currentPage,
+            ]}
```

Comment:
Here too, and in all the other tables, I'd move this dependency array up in a `const` to make it easier to maintain.

Reply:
same as above, shortened the list.